### DISCUSSION TITLE
SASS @use instead of @import

### DISCUSSION THREAD

**Discussion by TravisHi** [twbs/bootstrap#36050] [Q&A] [answered]
I was digging thru the docs and came across [https://getbootstrap.com/docs/5.1/customize/sass/#importing](https://getbootstrap.com/docs/5.1/customize/sass/#importing) where it uses `@import`. 

I'm not sure if this is CSS or SASS,  but if it's using SASS, the SASS docs indicate that `@use` should be used instead of `@import` [https://sass-lang.com/documentation/at-rules/import](https://sass-lang.com/documentation/at-rules/import).

It might be a good idea to explain why in the bootstrap 5 docs?

**Comment by mdo** [comment, marked as answer]
#29853 is where this will be tracked. Not going to be using that until v6.
